packages/cli/src/spec/configSpec.ts

```diff
@@ -17,22 +17,91 @@
 import {join} from 'path';
 import {homedir} from 'os';
 import {existsSync} from 'fs';
+import {promises as fsPromises} from 'fs';
 import {loadOrCreateConfig} from '../lib/config';
 import * as mock from 'mock-fs';
+import * as inquirer from 'inquirer';
+
+const DEFAULT_CONFIG_FOLDER = join(homedir(), '.bubblewrap-config');
+const DEFAULT_CONFIG_NAME = 'bubblewrap-config.json';
+const LEGACY_CONFIG_FOLDER = join(homedir(), '.llama-pack');
+const LEGACY_CONFIG_NAME = 'llama-pack-config.json';
+
+beforeAll(() => {
+  const fakeResult = Promise.resolve({
+    jdkPath: '/path/to/jdk',
+    androidSdkPath: '/path/to/android-sdk',
+  }) as any; // eslint-disable-line @typescript-eslint/no-explicit-any
+  spyOn(inquirer, 'prompt').and.returnValue(fakeResult);
+});
+
 
 describe('config', () => {
   describe('#loadOrCreateConfig', () => {
     it('checks if the file\'s name was changed in case it has the old name', async () => {
-      const dir = join(homedir(), '.llama-pack');
       // creates a mock file systes
       mock({
-        [dir]: {
+        [LEGACY_CONFIG_FOLDER]: {
           'llama-pack-config.json': '{}',
         }});
       await loadOrCreateConfig();
-      expect(existsSync(join(homedir(), '/.bubblewrap-config/bubblewrap-config.json'))).toBeTrue();
-      expect(existsSync(join(homedir(), '/.llama-pack/llama-pack-config.json'))).toBeFalse();
+      // checks if the file name was changed
+      expect(existsSync(join(DEFAULT_CONFIG_FOLDER, DEFAULT_CONFIG_NAME))).toBeTrue();
```

Comment:
We shouldn't have lines that are longer than 100 characters, could you break them up and make them neat please?